content/android/get-started.md

```diff
@@ -0,0 +1,202 @@
+
+Skip to the [**Download**](#download) section for directions on how to add WorldWind Android to your application. If you need some guidance with setting up the Android Studio IDE, the [**Setup**](#setup) section includes in-depth directions and walks through the creation of a sample project utilizing the WorldWind library.
+
+---
+### <a name="download"></a>Download
+
+For [Gradle](https://gradle.org) based projects, versioned WorldWind Android releases are hosted on [JCenter](https://bintray.com/bintray/jcenter). Grab the latest version by adding the following to the `dependencies` closure in your `build.gradle` file:
+```groovy
+compile 'gov.nasa.worldwind.android:worldwind:{{% latestBintrayVersion url="https://api.bintray.com/packages/nasaworldwind/maven/WorldWindAndroid/versions/_latest" %}}'
+```
+
+To manually download the (AAR) and other artifacts (source and javadocs) visit: https://bintray.com/nasaworldwind/maven/WorldWindAndroid
+
+`SNAPSHOT` builds are available through [OJO](https://oss.jfrog.org/). Add the following repository to your Gradle project:
+```groovy
+repositories {
+    maven {
+        url 'https://oss.jfrog.org/artifactory/oss-snapshot-local'
+    }
+}
+```
+Add the latest `SNAPSHOT` WorldWind Android dependency:
+```groovy
+compile 'gov.nasa.worldwind.android:worldwind:{{% latestOjoVersion url="https://oss.jfrog.org/artifactory/api/search/versions?g=gov.nasa.worldwind.android&a=worldwind&repos=oss-snapshot-local" %}}'
+```
+---
+### <a name="setup"></a>Setup
+#### Install Android Studio
+[Android Studio](https://developer.android.com/studio/index.html) is the IDE used by the WorldWind development team. It includes all the tools you need to build apps for Android. If you haven't done so already, download and install Android Studio.
+
+{{< button url="https://developer.android.com/studio/index.html" buttonText="Get Android Studio" >}}
+##### Installation Notes
+When you perform the installation you can choose the *Standard Setup*, which will download the required Android SDK components.
+##### Prerequisites
+Android Studio requires a compatible [Java SE Development Kit (JDK)](http://www.oracle.com/technetwork/java/javase/downloads/index.html) be installed on your computer. The installation instructions may prompt you to download and install an appropriate JDK (JDK 8 is recommended).
+#### Learning more about the Android Platform
+- Familiarize yourself with the [Android Developer's Site](http://developer.android.com/).
+- Read "[Introduction to Android](https://developer.android.com/guide/index.html)" and "[Application Fundamentals](https://developer.android.com/guide/components/fundamentals.html)" in the [Android Developer's Guide](http://developer.android.com/guide/).
+- Create a bookmark to the [Android API Docs](http://developer.android.com/reference/packages.html).
+
+#### How to build a basic WorldWind Android 'app'
```

Comment:
The layout doesn't seem to work very well in this section. I think refactoring it as a Tutorial would give you room to use a better layout by utilizing headings that you're avoiding now.

Again, be conservative with formatting in the middle of a sentence. We don't need to highlight the name of every code object. It's enough to use correct capitalization. The code blocks however, make perfect sense and look great.